src/xcode/redeclipse-Info.plist

```diff
@@ -5,9 +5,9 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleExecutable</key>
-	<string>redeclipse</string>
+	<string>blue-nebula</string>
```

Comment:
`src/xcode/redeclipse.xcodeproj/project.pbxproj` still references RE; can you remove the entire `xcode` folder, along with `src/macbuild.sh` as cmake is used?

`src/blue-nebula.cbp` could also be removed/edited

Reply:
`macbuild.sh` is gone already. I think we also won't need these frameworks anymore, they're annoying to work with anyway. The Code::Blocks project is gone, too.